config/file-dependency-updater/index.js

```diff
@@ -31,6 +31,27 @@ function removeFile(targetFilePath) {
     })
 }
 
+function isOSX() {
+    return process.platform === 'darwin';
+}
+
+function getOptions() {
+    const options = {
+        ignored: /(^|[\/\\])\../,
+        alwaysStat: true,
+        ignoreInitial: true
+    };
+    if (!isOSX()) {
```

Comment:
I meant, we might need to have a condition; `if (isLinux()) {` instead but I do not know yet. I am fine with your changes right.